java-client/src/main/java/org/opensearch/client/transport/httpclient5/ResponseException.java

```diff
@@ -64,7 +64,7 @@ static String buildMessage(Response response) throws IOException {
             response.getRequestLine().getMethod(),
             response.getHost(),
             response.getRequestLine().getUri(),
-            response.getStatusLine().toString()
+            response.getStatusLine()
```

Comment:
Let's pass `this.response.getStatusLine().getStatusCode()`? We don't need the full `StatusLine` here ...

Reply:
hm, that was just an unrelated cleanup (the `toString` happens automatically) which my IDE nicely suggested - i can change it to `getStatusCode()`, but that'd change the behaviour - if you're ok with that i can do it (though i'll split it out into a separate commit so that it's more obvious)

Reply:
> hm, that was just an unrelated cleanup

Oh I see, sure, let's use the `response.getStatusLine()`, apologies for confusion